src/tools/perf/lib/libperf.c

```diff
@@ -1557,6 +1563,8 @@ static ucs_status_t ucp_perf_test_setup_self_endpoints(ucx_perf_context_t *perf)
         if (status != UCS_OK) {
             goto err_destroy_eps;
         }
+
+        ucp_worker_release_address(worker, worker_attr.address);
```

Comment:
should we set address to NULL after we release it, to avoid double-free in line 1580?
(Maybe Coverity warning was justified?)

Reply:
Sounds good